### ISSUE TITLE
Let new players know about the F1 menu

### ISSUE THREAD

**Issue by 5URG3** [tsunamayo/Starship-EVO#4865]
New players ask about 1/8 scale, large block scales and grid offsets repeatedly on the official discord. 

Considering how useful it is, it is too obscure.

Please let players know that the F1 menu exists during the tutorial.

**Comment by Uncle-Ulty** [comment]
A Help tab would be great. A way to search for terms and know what they do

**Comment by ProPeach** [comment]
Another potential solution for the F1 menu being so obscure is to place a tab on the top of the Codex that takes you to the menu, same as the entity menu or clipboard menu. Keep the F1 key as a hockey to bring it up quickly, but if it's right there in the codex then far more players will find it purely by exploring the menu.

The settings in there are fundamental to the build experience, they need to be easily accessible and discoverable

**Comment by 5URG3** [comment]
Peach's option is also good. Works for me.

**Comment by cyberjasse** [comment]
or perhaps an additional step in the the building phase of the in-game tutorial ?

**Comment by tsunamayo** [comment]
@5URG3 Yes indeed it is on my to-do list, I wanted to add a step in the tutorial (better, do an advanced building tutorial). I want to add more keybind in the HUD also, but I also want to add a way to hide for advanced user.
I will also add a "learning center" inside the menu, with different pages to learn about various aspect of the game.
Do hesitate to tell me also what other trouble newcomers are having, it is really super super important and precious info for me (as they rarely post here).
Thanks!
